pkg/security/retry.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
+
+import (
+	"time"
+
+	retry "github.com/grpc-ecosystem/go-grpc-middleware/retry"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+
+	"istio.io/istio/security/pkg/monitoring"
+	"istio.io/pkg/log"
+)
+
+var caLog = log.RegisterScope("ca", "ca client", 0)
+
+// CARetryOptions returns the default retry options recommended for CA calls
+// This includes 5 retries, with backoff from 100ms -> 1.6s with jitter.
+var CARetryOptions = []retry.CallOption{
+	retry.WithMax(5),
```

Comment:
Not clear what is the current max attempts for google ca. @myidpt is 5 times sufficient?

Reply:
This matches existing retry options. Note if its down beyond 5 retries, its no big deal; we will send an error back to envoy and it will do its own indefinite retrying (with backoff)

Reply:
Make sense. Thanks for clarifying this.